### PR TITLE
Add optional statuses back

### PR DESCRIPTION
**Pull request by Ifropc**

### CODE REVIEW

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx [hunk 1]
```diff
@@ -552,6 +552,30 @@ To execute this update, simply run
...
 
+##### KYC Being Collected/Verified
+
+:::tip
+
+This step is optional. Most business don't use it: you can skip it and go to the [next one](#ready-to-receive-funds)
+
+:::
+
+Update for this action is simply changing transaction status. It's a good idea to modify message too, reflecting details of the process:
```
Comment (JakeUrban) [inline docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:563]: I would add that using this status makes the most sense when KYC verification may need to be performed asynchronously.

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx [hunk 2]
```diff
@@ -595,6 +619,31 @@ Asset field is optional. When not set, it defaults to the asset user requested.
...
 
+##### Waiting For User Funds
+
+:::tip
+
+This step is optional. Most business don't use it: you can skip it and go to the [next one](#sending-stellar-transaction)
+
+:::
+
+After update in the previous chapter has been made, user will be notified that your service is ready to receive funds. In real world, transfer confirmation may take time. In such cases, transaction should be set to a new status, after funds have been received, but not confirmed yet.
```
Comment (JakeUrban) [inline docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx:630]: I would add a note that the business should only use this status if the off-chain payment method being used notifies them of a pending inbound payment.

**Review by JakeUrban — APPROVED** [review]
LGTM, but left some suggestions

File: docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx [hunk 3]
```diff
@@ -552,6 +552,30 @@ To execute this update, simply run
 ./call-path.sh kyc-in-process.json
 ```
 
+##### KYC Being Collected/Verified
```
[the remark this answers is not in the document]
Comment (Ifropc) [reply docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx]: This section was already in the guide, I made a duplicate by mistake

**Pull request merged**